projects/packages/blaze/src/class-blaze.php

```diff
@@ -215,11 +215,14 @@ public static function jetpack_blaze_row_action( $post_actions, $post ) {
 	 */
 	public static function enqueue_block_editor_assets() {
 		/*
-		 * We do not want (nor need) Blaze in the site editor, or the widget editor, or the classic editor.
-		 * We only want it in the post editor.
-		 * Enqueueing the script in those editors would cause a fatal error.
-		 * See #20357 for more info.
-		 */
+		* We do not want (nor need) Blaze in the site editor, or the widget editor, or the classic editor.
+		* We only want it in the post editor.
+		* Enqueueing the script in those editors would cause a fatal error.
+		* See #20357 for more info.
+		*/
```

Comment:
The indenting here somehow got screwed up
```suggestion
		 * We do not want (nor need) Blaze in the site editor, or the widget editor, or the classic editor.
		 * We only want it in the post editor.
		 * Enqueueing the script in those editors would cause a fatal error.
		 * See #20357 for more info.
		 */
```

Reply:
PHPCBF fail. Thanks!